tests/Teams/MessagingExtensionAuth/TeamsAppManifest/manifest.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.5/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.5",
```

Comment:
>1.5 [](start = 22, length = 3)

I'm wondering if we're attempting to be mindful of what features are supported in what level?  Or just using latest?

Reply:
Currently, it looks like we have manifests in the scenarios using manifestVersion 1.3, 1.5 and one with devPreview.  I've asked Andrew what we should be using.  Waiting for a response.

Reply:
@daveta Andrew replied that manifestVersions should be 1.5, so I've included that as well.